src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1106,15 +1106,21 @@ bool PolicyHandler::ReceiveMessageFromSDK(const std::string& file,
                                           const BinaryMessage& pt_string) {
   POLICY_LIB_CHECK(false);
 
-  bool ret = policy_manager_->LoadPT(file, pt_string);
-  LOG4CXX_INFO(logger_, "Policy table is saved: " << std::boolalpha << ret);
-  if (ret) {
+  auto load_pt_result = policy_manager_->LoadPT(file, pt_string);
```

Comment:
i think `load_pt_result` should be const

Reply:
@mked-luxoft fixed in https://github.com/smartdevicelink/sdl_core/commit/d6e668e24d4444f10daf55c953b21af69c1be690